captum/attr/_utils/common.py

```diff
@@ -13,6 +13,14 @@ class ExpansionTypes(Enum):
     repeat_interleave = 2
 
 
+def safe_div(denom, quotient, default_value=None):
```

Comment:
should the default value be 0.0 here?

Reply:
The `default_value` is what is returned if `denom / quotient` is undefined, will document this